docs/release/new-release-series.md

```diff
@@ -33,85 +37,76 @@ Do first, anytime before the month of the release
 Do afterward, anytime before the month of the release
 -----------------------------------------------------
 
--   Create a blank `osg-3.4` SVN branch and add `buildsys-macros` package
-
-    1.  svn copy `buildsys-macros` from trunk and hand-edit it to hardcode all the new values
-
-        For EL 6:
+-   Create a blank `osg-3.X` SVN branch and add `buildsys-macros` package
 
-        1.  Edit the spec file and set `dver` to 6
-        2.  Run the following commands (adjust the NVR as necessary):
-
-                :::console
-                $ osg-build rpmbuild --el6
-                $ osg-koji import _build_results/buildsys-macros-*.el6.src.rpm
-                $ osg-koji import _build_results/buildsys-macros-*.el6.noarch.rpm
-                $ osg-koji tag-pkg osg-3.4-el6-development buildsys-macros-7-5.osg34.el6
+    1.  svn copy `buildsys-macros` from trunk and hand-edit it to hardcode the new `osg_version` and `dver` values
 
-        Then do the same for EL 7:
+        Do the following steps for all EL versions relevant to the new series; eg, for EL 7:
```

Comment:
```suggestion
        Do the following steps for all EL versions relevant to the new series; e.g., for EL 7:
```

Reply:
Personally i think it's legit to say "eg" without the dots, in the same way that it's normal to say "etc", rather than et. c.